nimbus-jose-jwt_aws-kms-extension/src/main/java/com/nimbusds/jose/aws/kms/crypto/KmsAsymmetricVerifier.java

```diff
@@ -0,0 +1,123 @@
+/*
+  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+
+  Licensed under the Apache License, Version 2.0 (the "License").
+  You may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+ */
+
+package com.nimbusds.jose.aws.kms.crypto;
+
+
+import com.amazonaws.services.kms.AWSKMS;
+import com.amazonaws.services.kms.model.DependencyTimeoutException;
+import com.amazonaws.services.kms.model.DisabledException;
+import com.amazonaws.services.kms.model.InvalidGrantTokenException;
+import com.amazonaws.services.kms.model.InvalidKeyUsageException;
+import com.amazonaws.services.kms.model.KMSInternalException;
+import com.amazonaws.services.kms.model.KMSInvalidSignatureException;
+import com.amazonaws.services.kms.model.KMSInvalidStateException;
+import com.amazonaws.services.kms.model.KeyUnavailableException;
+import com.amazonaws.services.kms.model.MessageType;
+import com.amazonaws.services.kms.model.NotFoundException;
+import com.amazonaws.services.kms.model.VerifyRequest;
+import com.amazonaws.services.kms.model.VerifyResult;
+import com.nimbusds.jose.CriticalHeaderParamsAware;
+import com.nimbusds.jose.JOSEException;
+import com.nimbusds.jose.JWSHeader;
+import com.nimbusds.jose.JWSVerifier;
+import com.nimbusds.jose.RemoteKeySourceException;
+import com.nimbusds.jose.aws.kms.crypto.impl.KmsAsymmetricSigningCryptoProvider;
+import com.nimbusds.jose.aws.kms.exceptions.TemporaryJOSEException;
+import com.nimbusds.jose.crypto.impl.CriticalHeaderParamsDeferral;
+import com.nimbusds.jose.util.Base64URL;
+import java.nio.ByteBuffer;
+import java.util.Set;
+import javax.annotation.concurrent.ThreadSafe;
+import lombok.NonNull;
+import lombok.var;
+
+/**
+ * Sign verifier implementation for RSA-SSA signing with public/private key stored in AWS KMS.
```

Comment:
[Nit] Same as above. Kindly add supported algorithms in comment.

Reply:
Fixed.